src/olympia/signing/tests/test_views.py

```diff
@@ -141,15 +141,15 @@ def test_new_addon_random_slug_unlisted_channel(self):
 
     def test_user_does_not_own_addon(self):
         self.user = UserProfile.objects.create(
```

Comment:
Certainly not a requirement for this patch but we should replace these `UserProfile.objects.create()` with `user_factory`

Reply:
I almost did but didn't want to dig into any potential repercussions just now